src/app/app.routes.ts

```diff
@@ -1,14 +1,39 @@
 import { Routes } from '@angular/router';
-import { RegisterComponent } from './components/register/register.component';
-import { LogicComponent } from './components/login/login.component';
+import { RegisterComponent } from './userManagement/components/register/register.component';
+import { LoginComponent } from './userManagement/components/login/login.component';
+import { HomeComponent } from './components/home/home.component';
+import { AuthGuard } from './userManagement/guards/auth.guard';
 
 export const routes: Routes = [
+  { path: 'login', component: LoginComponent, canActivate: [AuthGuard] },
   {
     path: 'register',
     component: RegisterComponent,
+    canDeactivate: [AuthGuard],
   },
   {
-    path: 'login',
-    component: LogicComponent,
+    path: 'trips',
+    component: HomeComponent,
+    canActivateChild: [AuthGuard],
+    // TO-DO: create the trip component
+    // children: [
+    //   {
+    //     path: 'trip-details/:id',
+    //     component: TripComponent,
+    //     canDeactivate: [AuthGuard],
+    //   },
+    // ],
+  },
+  {
+    path: '',
+    component: LoginComponent,
+    pathMatch: 'full',
+    canActivate: [AuthGuard],
+  },
+  {
+    path: '**',
+    component: LoginComponent,
+    pathMatch: 'full',
+    canActivate: [AuthGuard],
```

Comment:
Perhaps consider `redirectTo` or a`NotFoundPage`. 

Also, with regards to the `AuthGuard`, why is it required by your Login page or the wildcard route? 

Normally, `canActivate` is used for your Home page for example, but would this implementation not also prevent access to your Login page?

Reply:
Okay cool, i've changed to redirectTo and will do the NotFoundPage in the error handling PR.

Oh my `canActivate` function always returns true (for now). But when they have the app installed on the phone i would like for the user to be able to just go straight to the home page... thats why I implemented it that way.